outbound.js

```diff
@@ -1144,7 +1147,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
         }
 
         // TLS
-        if (smtp_properties.tls && cfg.enable_tls && !secured) {
+        if (!(self.todo.domain in tls_config.no_tls_hosts) &&
+            !(host in tls_config.no_tls_hosts) &&
+            smtp_properties.tls && cfg.enable_tls && !secured)
+        {
+            console.log(tls_config.no_tls_hosts);
```

Comment:
Yeah it works, but obviously no console.logs should be there :)

On Mon, Feb 22, 2016 at 4:15 PM, Matt Simerson <notifications@github.com>
wrote:

> In outbound.js
> <https://github.com/haraka/Haraka/pull/1340#discussion_r53694131>:
>
> > @@ -1144,7 +1147,12 @@ HMailItem.prototype.try_deliver_host_on_socket = function (mx, host, port, socke
> >          }
> >
> >          // TLS
> > -        if (smtp_properties.tls && cfg.enable_tls && !secured) {
> > +        if (!(self.todo.domain in tls_config.no_tls_hosts) &&
> > +            !(host in tls_config.no_tls_hosts) &&
> > +            smtp_properties.tls && cfg.enable_tls && !secured)
> > +        {
> > +            console.log(tls_config.no_tls_hosts);
>
> Please *read* the messages. It sure *looks* like @baudehlo
> <https://github.com/baudehlo> is still working on it. You'll also notice
> that it's in an -alpha release version, for early testing. Who was silly
> enough to claim production ready?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haraka/Haraka/pull/1340/files#r53694131>.
>